ui/apps/platform/src/hooks/usePermissions.ts

```diff
@@ -1,32 +1,30 @@
-import { useSelector } from 'react-redux';
-import { createStructuredSelector } from 'reselect';
+import { createContext, useContext } from 'react';
 
-import { selectors } from 'reducers';
-import type { Access } from 'types/role.proto';
 import type { ResourceName } from 'types/roleResources';
 import { replacedResourceMapping } from 'constants/accessControl';
+import { Access } from 'types/role.proto';
```

Comment:
Previously, this was `import type { Access } from 'types/role.proto';`. Should we keep the type-only import?

Reply:
👍 I keep missing this. I wonder if there is a VSCode setting that will prefer the `type` import variant when auto importing?